x/ibc/03-connection/client/cli/query.go

```diff
@@ -0,0 +1,109 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/ibc/03-connection/types"
+)
+
+// TODO: get proofs
```

Comment:
What needs to be done to enable the proof fetching?

Reply:
return an `ics23.Proof` on the query `res`